### PR TITLE
Label pull requests by area and from forks

### PR DESCRIPTION
**Pull request by aholstrup1**
Label PRs to look like this (for example)
image

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+"Area: Build":
```
Comment (mazhelez) [inline .github/labeler.yml]: .yaml instead?

File: .github/workflows/PullRequestLabeler.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        with:
+            repo-token: '${{ secrets.GITHUB_TOKEN }}'
+            sync-labels: true
+      
+      - name: Label community contribution
+        if: github.event.pull_request.head.repo.full_name != github.repository
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          gh api /repos/${{ github.repository }}/issues/${{ github.event.pull_request.number }}/labels  -f "labels[]=Community Contribution" -H "Accept: application/vnd.github.v3+json" -H "X-GitHub-Api-Version: 2022-11-28"
```
Comment (mazhelez) [inline .github/workflows/PullRequestLabeler.yaml]: How about "External Contribution"?

File: .github/labeler.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+- build/**
+- .github/**
+- .azuredevops/**
+
+"Area: System Application":
```
Comment (mazhelez) [inline .github/labeler.yml]: What about something like:

> "Build: AL-Go"
"Build: automation"

and 

> "AL: System Application"
"AL: Developer Tools"

or

> "App: System Application"
> "App: Developer Tools"

Comment (aholstrup1) [reply .github/labeler.yml]: I like that :)

File: .github/workflows/PullRequestLabeler.yaml (hunk 2 above)
Comment (JesperSchulz) [reply .github/workflows/PullRequestLabeler.yaml]: Let's discuss.

**Pull request merged**